### PR TITLE
TorsionDrive benchmark update

### PR DESCRIPTION
**Pull request by jthorton**
- [X] Created a new folder in the submissions directory containing the dataset
- [X] Added README.md describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [X] All files used to produce the dataset are included with a description
- [X] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [X] Dataset is named `dataset.json`
- [X] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [x] Ready to submit!

## Info
This dataset combines the theory benchmark torsiondrives made by @hyejang including the addition of some new charged molecules into a single dataset with multiple specs and will replace all theory benchmark sets. The inputs have been reused from past datasets which should ensure task deduplication.

### CODE REVIEW

File: submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/6-31+GSS ': {'method': 'B3LYP-D3BJ',
+  'basis': '6-31+gss',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/6-31+GSS ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/6-31+GSS ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/6-311+GSS ': {'method': 'B3LYP-D3BJ',
+  'basis': '6-311+gss',
```
Comment (dotsdl) [inline submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/README.md:109]: To verify: we have listed here `GSS` and `gss`, but in the submittable `dataset.json.bz2`, we have `G**` and `g**`, respectively. Do we know which one QCEngine will take?

**Review by dotsdl — APPROVED** [review]
Looks great @jthorton! I fixed trailing spaces in the `basis` key for each spec. Otherwise, I think we may be missing one of the input files. 

Since we're generating conformers, I suppose many of these torsiondrives may get recomputed? Not the biggest deal, since at the moment we have the compute, but wasn't obvious to me that the approach would give the same inputs.

File: submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/README.md (hunk 1 above)
Comment (dotsdl) [reply submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/README.md:109]: From discussion: our submission includes `g**`, which should be what we want. The QCSubmit validation will convert these to `gss` for validation with basis-set-exchange, but push through `g**` for `psi4` execution.

File: submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/QCSubmit_workflow.ipynb [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+      "/Users/joshua/Documents/Software/qcsubmit/qcsubmit/validators.py:301: UserWarning: The molecule C13H12N3O has a net charge of -1.0.\n",
+      "  UserWarning,\n"
+     ]
+    }
+   ],
+   "source": [
+    "# now create the dataset from the selected_mol2s, containing mol2 files of additional charged molecules\n",
+    "dataset = factory.create_dataset(\n",
+    "    dataset_name=\"OpenFF Theory Benchmarking Set v1.0\",\n",
+    "    molecules='selected_mol2s',\n",
```
[the remark this answers is not in the document]
Comment (jthorton) [reply submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/QCSubmit_workflow.ipynb:100]: Good catch this should now be there.

**Review by dotsdl — APPROVED** [review]
Thanks @jthorton!

**Comment by dotsdl** [comment]
Merging!

**Comment by dotsdl** [comment]
I know what happened here. This submission featured a file called `dataset 2.json`, which due to recent changes in our globbing pattern for submittable datasets to `dataset*.json*` from `dataset.json*`, was being picked up and handled by the automation.

I have changed the `dataset 2.json` file to `refdataset.json` in `master`. This *should* avoid the automation seeing it at all, even though it is still a listed file in this PR (we filter for actually-present files).

**Comment by dotsdl** [comment]
I have also moved this submission back to error cycling; it should proceed accordingly.

**Comment by dotsdl** [comment]
Setting this to "Requires Scientific Review". No additional forward progress is being made on the remaining incomplete torsiondrives due to what now look like systematic errors.

@hyejang if the remaining errors are acceptable to you, we can also move to "Archived/Complete".

**Comment by pavankum** [comment]
@dotsdl Sorry missed this, will check and get back to you

**Pull request merged**